index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <link href="style/style.css" type="text/css" rel="stylesheet">
+        <title>Сurriculum vitae</title>
+    </head>
+    <body>
+        <header>
+            <section class="info">
+                <h1>Dmitry Kravstov</h1>
+                <div class="contact">
+                    <p>Telephone Number:<br><br>Email:</p>
+                </div>
+                <div class="contact_info">
+                    <p>+375293908345<br><br>Zerover06@gmail.com</p>
+                </div>
+            </section>
+            <section class="image">
+                <img src="picture/face.jpg" width="250" height="200" alt="face">
+            </section>
+        </header>
+        <section class="Summary">
+            <h1>Summary</h1>
+            <p> Programming has always been around me, Pascal, C ++, Java, I always liked it and always approached the fulfillment of tasks with eagerness. Now I decided to start a junior Front-end developer career and adapt to it all the time and effort.<br> I have good analytical skills. Easy to learn. I work well in a team.</p>
+
+        </section>
+        <section class="Skills">
+            <h1>Skills</h1>
+            <div class="Skill_name">
+                    <p>JavaScript/CSS3/HTML5:<br><br>Windows:<br><br>Git:<br><br>English:</p>
+                </div>
+                <div class="Skill_info">
```

Comment:
you were advised to follow the guideline http://codeguide.academy/html-css.html, that recommends to use '-' for separating words in class names "skill-name"